src/frontend/src/utils/validateAlias.ts

```diff
@@ -1,29 +0,0 @@
-// Regex Pattern for input: All characters, must be alphabet or number
-// Can have hyphen(s), space(s) or underscore(s) in the middle.
-// Good examples: "2019_macbook", "2019-Macbook", "2019 Macbook"
-// Bad examples: "2019 macbook!", "2010 macbook_", "space trails at end "
-export const validateAlias = (
```

Comment:
Given this got even simpler and was not used anywhere else I moved the function into the alias component.